frontend/packages/shared/src/components/InputActionWrapper/InputActionWrapper.test.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { fireEvent, render as rtlRender, screen } from '@testing-library/react';
+import type { InputActionWrapperProps } from './InputActionWrapper';
+import { InputActionWrapper } from './InputActionWrapper';
+
+jest.mock('./InputActionWrapper', () => ({
+  __esModule: true,
+  InputActionWrapper: ({ children, onEditClick, onDeleteClick, onSaveClick }: any) => (
+    <div>
+      {children}
+      <button onClick={onEditClick} aria-label='general.edit'>
+        Edit
+      </button>
+      <button onClick={onSaveClick} aria-label='general.save'>
+        Save
+      </button>
+      <button onClick={onDeleteClick} aria-label='general.delete'>
+        Delete
+      </button>
+    </div>
+  ),
+}));
+
+describe('InputActionWrapper', () => {
+  const mockProps: InputActionWrapperProps = {
+    children: <div />,
+    mode: 'standBy',
+    onEditClick: jest.fn(),
+    onDeleteClick: jest.fn(),
+    onSaveClick: jest.fn(),
+  };
+
+  it('render save buttont', () => {
+    render(mockProps);
+    const button = screen.getByRole('button', { name: 'general.save' });
+    expect(button).toBeInTheDocument();
+  });
+
+  it('render delete buttont', () => {
+    render(mockProps);
+    const button = screen.getByRole('button', { name: 'general.delete' });
+    expect(button).toBeInTheDocument();
+  });
+
+  it('render edite buttont', () => {
+    render(mockProps);
+    const button = screen.getByRole('button', { name: 'general.edit' });
+    expect(button).toBeInTheDocument();
+  });
+
+  it('handles focus and blur events', () => {
+    render(mockProps);
+    const wrapperSave = screen.getByRole('button', { name: 'general.save' });
+    const wrapperEdit = screen.getByRole('button', { name: 'general.edit' });
+    const wrapperDelete = screen.getByRole('button', { name: 'general.delete' });
+
+    fireEvent.focus(wrapperEdit);
```

Comment:
We should consider switching from fireEvent to userEvent since userEvent is the recommended choice by the React Testing Library itself. 😊